### ISSUE TITLE
Finish Scope

### ISSUE THREAD

**Issue by NoahS2003** [AceAttorneyMaster111/St.-Johns-Sorting#1]
Our scope is detailed in the README file. Before adding on extra features, this should be finished. Other bugs, however, take precedence over new code, even contributing to the scope.

**Comment by griffindugan** [comment]
What do you mean "scope"? Do you mean the example and what we have to do? Do you mean that the README isn't finished? And has to be before we start coding?

@AceAttorneyMaster111

**Comment by NoahS2003** [comment]
No. I mean that we shouldn't add extra features until we finish the requirements.
